src/testflinger_device_connectors/devices/muxpi/muxpi.py

```diff
@@ -313,6 +313,15 @@ def create_user(self, image_type):
                 self._run_control(cmd)
                 self._configure_sudo()
             if image_type == "tegra":
+                base = self.mount_point / "writable"
+                ci_path = base / "var/lib/cloud/seed/nocloud"
+                self._run_control(f"sudo mkdir -p {ci_path}")
+                self._run_control("mkdir -p {}".format(remote_tmp))
```

Comment:
any reason not to use fstrings here? I know we probably still have a lot of remnants of .format() throughout this code because we used to use it on a much older python version, but you can also see that we are starting to adopt fstrings now too, because everything we deploy on now is 3.8+ You even used fstrings in other parts of this patch :)

Reply:
Thanks for the review. I just copy pasted the same stanza which was used already in 4 other places. Should I take the opportunity to align them as well? Otherwise I can just change this one.